src/extension/importer/core/src/main/java/org/geoserver/importer/Importer.java

```diff
@@ -1339,6 +1339,7 @@ void loadIntoDataStore(
             featureType = featureDataConverter.convertType(featureType, format, data, task);
             UpdateMode updateMode = task.getUpdateMode();
             final String uniquifiedFeatureTypeName;
+            boolean newSchema = false;
```

Comment:
Isn't used anywhere?

Reply:
Right, PMD complained and I fixed it... but the change was left on my local disk :-D